pyglmnet/utils.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+
+def softmax(w):
+    """Softmax function of given array of number w
+
+    Parameters
+    ----------
+    w: array | list
+        The array of numbers.
+
+    Returns
+    -------
+    dist: array
+        The resulting array with values ranging from 0 to 1.
+    """
+    w = np.array(w)
+    maxes = np.amax(w, axis=1)
+    maxes = maxes.reshape(maxes.shape[0], 1)
+    e = np.exp(w - maxes)
+    dist = e / np.sum(e, axis=1, keepdims=True)
+    return dist
+
+
+def label_binarizer(y):
+    """Mimics scikit learn's LabelBinarizer
+
+
+    Parameters
+    ----------
+    y: ndarray (n_samples)
```

Comment:
you mean the shape is `(n_samples)`? Two comments here: you need to specify that it's the shape. Also, shape is a tuple, so you need to say `(n_samples, )`. Compare:

```py
>>> x = (100)
>>> type(x)
```
and 

```py
>>> x = (100, )
>>> type(x)
```